src/test/java/vehicle/CVSB_3845/VehicleDetails_CVSB_6629.java

```diff
@@ -72,11 +72,11 @@ public void testAllDetailsHgv() {
         hgvDetailsSteps.checkYearOfManufactureIs("2018");
         hgvDetailsSteps.checkDateOfFirstRegistrationIs("24/06/2019");
         hgvDetailsSteps.checkAxleIs("3");
-        hgvDetailsSteps.checkDtpIs("178202");
+        hgvDetailsSteps.checkDtpIs("sdgs");
```

Comment:
Why using random string here?

Reply:
so it is defined in the seeded file.. test only checks the correct value is returned